packages/office-ui-fabric-react/src/components/ColorPicker/ColorRectangle/ColorRectangle.base.tsx

```diff
@@ -168,14 +168,6 @@ export class ColorRectangleBase extends React.Component<IColorRectangleProps, IC
       return;
     }
 
-    // If the primary button (1) isn't pressed, the user is no longer dragging, so turn off the
```

Comment:
If you could leave this there but commented in case it's needed in the future for debugging purposes, that would potentially be helpful. Something like this:
```
    // Leaving the following commented code which is sometimes necessary for debugging:
    // If the primary button (1) isn't pressed, the user is no longer dragging, so turn off
    // the event handlers and exit.
    // tslint:disable-next-line:no-bitwise
    // if (!(ev.buttons & 1)) {
    //   this._disableEvents();
    //   return;
    // }
```

Reply:
Done!